satellite/repair/checker/checker.go

```diff
@@ -55,6 +56,7 @@ type Checker struct {
 	metainfo        *metainfo.Service
 	metaLoop        *metainfo.Loop
 	nodestate       *ReliabilityCache
+	overrideRepair  int32
```

Comment:
`repairOverride` instead of `overrideRepair` for consistency with the config value name. I have the same suggestion for `checkerObserver` and `SegmentRepairer`